frontend/app/src/components/status/sync/SyncButtons.vue

```diff
@@ -24,7 +24,7 @@ const action = (action: SyncAction) => {
 
 <template>
   <div class="flex flex-row justify-between gap-1">
-    <RuiTooltip open-delay="400" class="w-full" close-delay="50000">
+    <RuiTooltip open-delay="400" class="w-full">
```

Comment:
I'd like to know who responsible for this.🤪

Reply:
@lukicenturi hahahaha hmm probably me, if you do git-blame I am pretty sure my name is there.

I tried something and I supposedly reverted it to remove the close-delay...